internal/storage/sbom_store_test.go

```diff
@@ -0,0 +1,348 @@
+package storage
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/apiserver/pkg/storage"
+	"k8s.io/utils/ptr"
+
+	"github.com/jmoiron/sqlx"
+	_ "modernc.org/sqlite"
+
+	"github.com/rancher/sbombastic/api/storage/v1alpha1"
+)
+
+const keyPrefix = "/storage.sbombastic.rancher.io/sboms"
+
+type sbomStoreTestSuite struct {
+	suite.Suite
+	store       *sbomStore
+	db          *sqlx.DB
+	broadcaster *watch.Broadcaster
+}
+
+func (suite *sbomStoreTestSuite) SetupTest() {
+	suite.db = sqlx.MustConnect("sqlite", ":memory:")
+
+	suite.db.MustExec(CreateSBOMTableSQL)
+
+	suite.broadcaster = watch.NewBroadcaster(1000, watch.WaitIfChannelFull)
+	suite.store = &sbomStore{
+		broadcaster: suite.broadcaster,
+		db:          suite.db,
+	}
+}
+
+func (suite *sbomStoreTestSuite) TearDownTest() {
+	suite.db.Close()
+	suite.broadcaster.Shutdown()
+}
+
+func TestSBOMStoreTestSuite(t *testing.T) {
+	suite.Run(t, &sbomStoreTestSuite{})
+}
+
+func (suite *sbomStoreTestSuite) TestCreate() {
+	sbom := &v1alpha1.SBOM{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test",
+			Namespace: "default",
+		},
+	}
```

Comment:
Silly requirement, can you add also some labels, annotations and the `spec`? I know the `Spec` is going to change soon, but at least we can test the whole object